### DISCUSSION TITLE
Create linux repository for update

### DISCUSSION THREAD

**Discussion by Ilia1** [LizardByte/community#86] [Sunshine]
### Describe the Bug

Publish app for ubuntu in https://launchpad.net or create self-hosted linux repository.

### Host Operating System

Linux

### Operating System Version

ubuntu 22.04

### Architecture

64 bit

### Sunshine Version

0.14.1

### GPU Type

amd

### GPU Model

rx570

### GPU Driver/Mesa Version

19.1.0

**Comment by ReenigneArcher** [comment]
We're considering publishing packages to https://cloudsmith.io/~lizardbyte/repos

I will update this discussion once that happens.
